dev_guide/deployments.adoc

```diff
@@ -464,6 +501,54 @@ capacity (an in-place update).
 - `*maxUnavailable*=10%` and `*maxSurge*=10%` scales up and down quickly with
 some potential for capacity loss.
 
+Generally, if you want fast rollouts, use *maxSurge*, if you need to take into
+account resource quota and can accept partial unavailability, use *maxUnavailable*.
+
+=== Rolling example
+
+Rolling deployments are the default in {product-title}. To see a rolling update, follow these steps:
+
+1. Create a application based on the example deployment images:
```

Comment:
what is "example deployment images" ? AFAIK, `oc new app openshift/deployment-example` will instantiate a template with that name.